test/app/renderer/reducers/wallet.spec.ts

```diff
@@ -11,18 +10,9 @@ describe("Wallet renderer", () => {
     expect(newState).toMatchObject(newWallet)
   })
 
-  it("should not update empty state with invalid action", async () => {
```

Comment:
it's great to have these tests removed because of the improvements in the actions typings :clap: